samples/MerQure.Tools.Samples/Startup.cs

```diff
@@ -0,0 +1,39 @@
+﻿using MerQure.RbMQ;
+using MerQure.RbMQ.Config;
+using MerQure.Tools.Samples.RetryBusExample.Domain;
+using MerQure.Tools.Samples.RetryBusExample.RetryExchangeExample.Infra;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.IO;
+
+namespace MerQure.Tools.Samples
+{
+	public class Startup
+    {
+        public IConfiguration Configuration { get; set; }
+
+        public Startup()
+        {
+			var p = Directory.GetCurrentDirectory();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'p' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)